bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java

```diff
@@ -138,7 +139,10 @@ public void printPDF(final IXmlSerializable object, final PrintSettings printSet
          // setup XSL stylesheet source
          final StreamSource xslSource = new StreamSource(xslFile);
 
-         final Transformer transformer = TransformerFactory.newInstance().newTransformer(xslSource);
+         final TransformerFactory transformerFactory = TransformerFactory.newInstance();
+         transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, ""); //$NON-NLS-1$
+         transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, ""); //$NON-NLS-1$

```

Comment:
Replace `""` with `UI.Empty`

This is the only tiny issue which I found, except that the last `main` branch was not merged

Reply:
Fixed